### DISCUSSION TITLE
Resize windows to greater extents than default?

### DISCUSSION THREAD

**Discussion by mawil583** [rxhanson/Rectangle#1060] [General]
Under Keyboard Shortcuts > Size, you have options for Larger and Smaller window sizes. Love this feature! But I'd like to customize it a bit more. I find that I need use these shortcuts several times in a row before my window reaches the desired size. I'd like to have the shortcut resize my window to a greater degree than your default. Is that possible? If not, I'd like to suggest that as a new feature.

Speaking of which... Is this github issues tab the best place to make suggestions for new features?

Thanks!

**Comment by rxhanson** [comment]
Here's the terminal command for adjusting the size increments for make smaller/larger:
https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#make-smallermake-larger-size-increments

Restart the app after executing a terminal command.

For feature requests, just make a new Discussion. I'm going to convert this issue to a discussion.

About features:
Rectangle is intentionally slim, so I typically don't add more features into it at this point. If you have a feature that you really want to add in, I usually will accept good pull requests as long as the feature is added in as a Terminal command first, not in the UI. Let me know before you start a feature in case it happens to be something that I wouldn't merge a PR for. Rectangle Pro (paid) is where I've added a lot of things that didn't quite fit in Rectangle. I always love to hear new ideas, though, so fire away!
